tests/test_resources/test_project.yml

```diff
@@ -17,6 +17,8 @@ build:
 dependencies:
   build:
     - 'test/docker/'
+  deploy:
```

Comment:
Add a test that the test-only dependencies still work (they do not trigger a deployment)